Remove workaround for testing with PHP 8

greg0ire opened the pull request:
Now that all dependencies support it, it should not be necessary to
pretend we are using PHP 7.4. Implies upgrading the CS lib.

greg0ire committed:
Remove workaround for testing with PHP 8

Now that all dependencies support it, it should not be necessary to
pretend we are using PHP 7.4.

greg0ire committed:
Upgrade coding standard library

We should use a version that is compatible with PHP 8

SenseException commented on Tests/Command/LoadDataFixturesDoctrineCommandTest.php:
```diff
@@ -29,11 +31,13 @@ public function testInstantiatingWithoutManagerRegistry() : void
                 $this->expectExceptionMessage(
                     <<<'MESSAGE'
-Doctrine\Bundle\DoctrineBundle\Command\DoctrineCommand::__construct(): Argument #1 ($doctrine) must be of type Doctrine\Persistence\ManagerRegistry, null given, called in /home/runner/work/DoctrineFixturesBundle/DoctrineFixturesBundle/Command/LoadDataFixturesDoctrineCommand.php on line 49
+Doctrine\Bundle\DoctrineBundle\Command\DoctrineCommand::__construct(): Argument #1 ($doctrine) must be of type Doctrine\Persistence\ManagerRegistry, null given, called in /home/runner/work/DoctrineFixturesBundle/DoctrineFixturesBundle/Command/LoadDataFixturesDoctrineCommand.php on line 51
```
Is the line number relevant for the error message check? It seems like an unstable test when changes are happening.

greg0ire replied:
It is unstable indeed, I remember having to change it in the past. Let me try refactoring it.

greg0ire committed:
Refactor test to make it less fragile

We no longer assert the file and the line number in the test, which
should make it less flaky. Also, we regroup together the 2 cases which
makes it easier to understand the only thing that changes is the form of
the message we are trying to test.

greg0ire commented:
It looks like the integration tests are flaky :facepalm: 

```
 There was 1 error:

1) Doctrine\Bundle\FixturesBundle\Tests\IntegrationTest::testRunCommandWithPurgeMode
Symfony\Component\DependencyInjection\Exception\InvalidArgumentException: The "doctrine" service is private, you cannot replace it.

/home/runner/work/DoctrineFixturesBundle/DoctrineFixturesBundle/vendor/symfony/dependency-injection/Container.php:169
/home/runner/work/DoctrineFixturesBundle/DoctrineFixturesBundle/Tests/IntegrationTest.php:483
```
